Sources/SwiftParser/Expressions.swift

```diff
@@ -18,6 +18,11 @@ extension TokenConsumer {
     case (.awaitTryMove, let handle)?:
       var backtrack = self.lookahead()
       backtrack.eat(handle)
+
+      // These can be parsed as expressions with try/await.
+      if backtrack.at(anyIn: IfOrSwitch.self) != nil {
+        return true
+      }
```

Comment:
Am I missing something or does `atStartOfExpression` return `false` for `if` expressions?

Reply:
Good catch! Though it currently doesn't make a difference as `atStartOfExpression` is currently only be used to distinguish expressions from statements in brace element position, and we prefer to parse `if`/`switch` as statements there. We ought to still handle the case tho.